api/src/main/java/org/pine/api/PineRuntime.java

```diff
@@ -7,4 +7,6 @@
   PineLiker newLiker();
 
   void connect(String address);
+
+  void select(int index);
```

Comment:
What does this method do?

Reply:
Users can select the database stored in redis.
So I added this in the abstract method